hucssley-classes.md

```diff
@@ -867,7 +877,7 @@ $hu-content-alias: "";
 $hu-content-modules: (base);
 
 $hu-content-pseudos: (
-  "::after", 
+  "::after",
```

Comment:
Strange how this thinks it's changed after your previous one was merged… 🤔

Reply:
Maybe something to do with the settings fumbled around with to avoid adding new lines on save, I think?